### PR TITLE
Allow users to prefix searches with `_.`

### PR DESCRIPTION
**Pull request by captbaritone**

### CODE REVIEW

File: docs/main.js [hunk 1]
```diff
@@ -3,6 +3,10 @@
   var sections = document.querySelectorAll('.searchable_section');
   var searchInput = document.getElementById('function_filter');
 
+  function searchValue() {
+    return searchInput.value.trim().replace(/^_\./, '');
```
Comment (akre54) [inline docs/main.js]: How about `/^_\.?/` so you don't need to type the dot? Otherwise lgtm

Comment (captbaritone) [inline docs/main.js]: Updated.

**Comment by captbaritone** [comment]
I'll cherry-pick this over.

**Comment by captbaritone** [comment]
Done.

**Pull request merged**